Ship_Game/Universe/UniverseScreen/UniverseScreen.InputActions.cs

```diff
@@ -37,14 +37,6 @@ public void DoExplore(Ship s)
             s.AI.OrderExplore();
         }
 
-        public void DoDefense(Ship s)

```

Comment:
So if all of this code was broken, what does the AI even do to protect its Empire?

Reply:
When the AI detects a fleet coming in to a system, It creates a defense fleet. The defense fleet can also summon bombers and troops if the AI planet was taken - it will first clear any defense in the system and then get bombers and troops, So it can retake lost assets. The AI can also cancel other fleet tasks in order to create defense tasks. It is all part of the war logic I redesigned quite some time ago.